src/Components/SmartComponents/HybridInventory/ImmutableDevicesTab/helpers.test.js

```diff
@@ -33,4 +34,46 @@ describe('useOnLoad', () => {
             }
         );
     });
+});
+
+const defaultColumns = [
+    { key: 'updated', renderFunc: jest.fn() },
+    { key: 'system_profile', props: { width: 15 } },
+    { key: 'groups', props: { width: 15 } },
+  ];
+  
+  describe('mergeAppColumns', () => {
+    test('Should replace default OS column to vulnerabilty OS column', () => {
+      const result = mergeAppColumns(defaultColumns);
+  
+      const osColumn = result.find(
+        (column) => column.key === 'os'
+      );
+      expect(osColumn).toEqual(SYSTEMS_EXPOSED_HEADER.find((column) => column.key === 'os'));
+    });
+    test('Should disable sorting when edge groups are enforced', () => {
+        const enforce_edge_groups = true;
+        const result = mergeAppColumns(defaultColumns, enforce_edge_groups);
+    
+        const groupsColumn = result.find(
+          (column) => column.key === 'groups'
+        );
+        expect(groupsColumn).toEqual(expect.objectContaining({ props: { width: 15, isStatic: true } }));
```

Comment:
nit-pick: according to test desc, this should only `isStatic` value. So I suggest to not to check for width. This can create some confusion once we change the width and the test which checks for sorting disabled fails

Reply:
Hm, I think it is better to update the test description. Because I want to make sure that disabling the sorting happens by extending the props with `isStatic: true`

Reply:
Hmm, I think it is better to update the test description. Because I want to make sure that disabling the sorting happens by extending the props with `isStatic: true`, not mutating the props entirely